r2/r2/controllers/api.py

```diff
@@ -870,6 +873,7 @@ def POST_unfriend(self, nuser, iuser, container, type):
         if type in ("friend", "enemy") and container != c.user:
             abort(403, 'forbidden')
         fn = getattr(container, 'remove_' + type)
+
```

Comment:
Nit: please remove the newline being added here. We try to avoid making whitespace changes except when we're already adjusting code for another reason, and this is separated from the actual changes.